src/govuk/components/tabs/tabs.mjs

```diff
@@ -219,29 +219,37 @@ Tabs.prototype.onTabKeydown = function (e) {
 
 Tabs.prototype.activateNextTab = function () {
   var currentTab = this.getCurrentTab()
-  var nextTabListItem = currentTab.parentNode.nextElementSibling
-  if (nextTabListItem) {
-    var nextTab = nextTabListItem.querySelector('.govuk-tabs__tab')
-  }
-  if (nextTab) {
-    this.hideTab(currentTab)
-    this.showTab(nextTab)
-    nextTab.focus()
-    this.createHistoryEntry(nextTab)
+
+  if (currentTab.parentNode instanceof Element) {
```

Comment:
Look at using `currentTab.parentElement` instead as then we don't need the type check.

Reply:
Nice tip, I've pushed it up